src/Symfony/Component/Messenger/EventListener/SendFailedMessageToFailureTransportListener.php

```diff
@@ -42,6 +41,10 @@ public function onMessageFailed(WorkerMessageFailedEvent $event)
             return;
         }
 
+        if (!isset($this->failureSenders[$event->getReceiverName()])) {
+            return;
```

Comment:
You should throw an error here. Because you `array_fill` in the wiring, this should be all good 👍

Reply:
I changed the logic now. Maybe throwing an error doesn't make sense now.